.github/workflows/runtime-image-update.yaml

```diff
@@ -39,7 +42,7 @@ jobs:
         id: get_latest_tag
         # yamllint disable rule:line-length
         run: |
-          LATEST_TAG=$(python ./.github/scripts/get_latest_runtime_image_tag.py ${{ github.event.inputs.quay_repo }})
+          LATEST_TAG=$(python ../.github/scripts/get_latest_runtime_image_tag.py ${{ github.event.inputs.quay_repo }})
```

Comment:
Look like we only use this file in python-sdk. Should we move it there?

Reply:
moved it at `python-sdk/dev/integration_test_scripts`